index.js

```diff
@@ -5,6 +5,7 @@ module.exports = {
   NIO: 'Named Information Officer',
   NVS: 'Named Veterinary Surgeon',
   NTCO: 'Named Training and Competency Officer',
+  NPRC: 'Named Person Responsible for ensuring Compliance',
```

Comment:
`Ensuring` the lack of uppercase e makes me twitch.

Reply:
heh I debated it but it's not part of the acronym :D

Reply:
Paging Dr @pcuk1979

Reply:
Full role title is Named Person Responsible for Compliance according to the blue ASRU bible (https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/662364/Guidance_on_the_Operation_of_ASPA.pdf).

No 'ensuring'.